teuthology/suite/__init__.py

```diff
@@ -48,6 +49,8 @@ def process_args(args):
                 value = []
             else:
                 value = [x.strip() for x in value.split(',')]
+        elif key == 'validate_sha1':
```

Comment:
Suggestion to change the line to:
```
elif key in ('validate_sha1'):
```

Reply:
Sure - I followed the examples above where there was only a single key, but makes sense to use the set operation in case there are other bool options like this in the future. Done.